t/page/countries.rb

```diff
@@ -0,0 +1,26 @@
+require 'minitest/autorun'
+require_relative '../../lib/page/countries'
+require 'pry'
+
+CJSON = 'https://cdn.rawgit.com/everypolitician/everypolitician-data/%s/countries.json'.freeze
+SHA   = 'd8a4682f'.freeze
+
+describe 'Countries' do
+  subject do
+    Page::Countries.new
+  end
+
+  it 'has a list of all countries in Everypolitician' do
+    assert_equal subject.countries.first[:name], 'Abkhazia'
```

Comment:
Now that this is a Country object from `everypolitician-ruby`, I think it probably would be better to test `.name` rather than `:name`, yes (and probably do likewise in the template).